tools/codechecker_report_hash/.gitignore

```diff
@@ -0,0 +1 @@
+codechecker_report_hash.egg-info
```

Comment:
I think `build` and `dist` directories should be ignored too similar as in [tu_collector](https://github.com/Ericsson/codechecker/blob/master/tools/tu_collector/.gitignore).